### DISCUSSION TITLE
Change "Figure" prefix to "Title" and place caption above for ONE image

### DISCUSSION THREAD

**Discussion by eteitelbaum** [quarto-dev/quarto-cli#6325] [Q&A] [answered]
### Description

I am writing a paper as a .pdf document in Quarto.

I have a situation where I produced a table awhile back and exported it to a .png. Rather than including all of the code for the table in my Quarto document, I would like to just use the image, e.g. 

`![Summary Statistics](tables/table1.png)`

But of course when I do that, Quarto automatically labels it as a Figure. 

I know that if I want to change the caption prefix and location, I can use these options in the YAML header: 

```
fig-cap-location: top
crossref:
  fig-title: Table
```

But I only want to control the prefix and location of the caption for this one table. Then I want to switch back to the default prefix and location for the remaining figures in the document. 

I would also like to make sure that the numbering restarts at "Figure 1" for subsequent figures. 

Is there any way to have this kind of fine-grained control over figure captions? I was thinking adding arguments after calling the figure in {} might work like: 

`![Summary Statistics](tables/table1.png){fig-cap-location=top}`

But I couldn't get anything to work.

**Comment by mcanouil** [comment, marked as answer]
You can't right now (or use raw LaTeX).
Watch <https://github.com/quarto-dev/quarto-cli/issues/4944>.

**Comment by TC-RAC** [comment]
I am trying to do the same - I am currently trying to create a table using kableExtra and ten using column_spec(image=....) to put the image in there.... I am not sure I am winning though.... so if anyone else thinks this may be a viable work round and has suggestions - I too would be happy tot hear!

**Reply by mcanouil** [reply]
I don't follow, what does this have to do with the question here about figure [custom cross-reference](https://quarto.org/docs/authoring/cross-references-custom.html)?
Please open a new discussion and provide a small reproducible example to work with.
